src/Console/Output/Progress/ProgressOutputFactory.php

```diff
@@ -21,6 +21,15 @@
  */
 final class ProgressOutputFactory
 {
+    /**
+     * @var array<string, class-string>
```

Comment:
```suggestion
     * @var array<string, class-string<ProgressOutputInterface>>
```

if I remember right, it shall be possible